api/src/modules/navigations/controllers/admin-navigations.controller.ts

```diff
@@ -54,7 +54,7 @@ export class AdminNavigationsController {
 			await this.adminNavigationsService.findAllNavigationBarItems();
 
 		// filter based on logged in / logged out
-		const allowedUserGroups = user
+		const allowedUserGroups = user.getId()
```

Comment:
maybe group id is more logical?
```suggestion
		const allowedUserGroups = user.getGroupId()
```